<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sniffer</class>
 <widget class="QWidget" name="Sniffer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sniffer</string>
  </property>
  <widget class="QPlainTextEdit" name="Sniff_Text_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Sniff_But_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>426</y>
     <width>841</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QTableView" name="TableSrc">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>211</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="TableDst">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>211</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="TableProto">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>151</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
